src/java.base/share/classes/java/util/Locale.java

```diff
@@ -258,6 +259,74 @@
  * locales. For example, {@code Locale.US} is the {@code Locale} object
  * for the United States.
  *
+ * <h3><a id="default_locale">Default Locale</a></h3>
+ *
+ * <p>The default Locale is mainly provided for any locale-sensitive methods if no
```

Comment:
The word "mainly" can be omitted, it doesn't add any clarity.

Reply:
Removed.